testing/citest/README.md

```diff
@@ -0,0 +1,280 @@
+# Summary
+
+These Spinnaker integration tests depend on the [Cloud Integration Testing]
+(https://github.com/google/citest) python package. You'll need to run the
+setup script to install the dependencies into your environment, then can
+run the tests at will.
+
+To write new tests refer to existing tests and see
+[the citest README.md](https://github.com/google/citest/blob/master/README.md)
+gifor the time being, and ask questions on the [Spinnaker #dev slack channel]
```

Comment:
I'm going to fix this in a different CL once this goes in. Given all the history I'm moving around across repositories I want to minimize disturbances. I have some other changes (committed to the repo) while this PR has been in limbo that need to be propagated as well that I'll apply as well.